### PR TITLE
First version of the Dockerfile.

### PR DESCRIPTION
**Pull request by tmooney**
Includes toil and various tools required for alignment.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    nodejs \
+    python-dev \
+    python-pip \
+    tzdata \
+    unzip \
+    wget \
+    zlib1g-dev
+
+##############
+#Picard 2.9.0#
```
Comment (jasonwalker80) [inline Dockerfile]: I think we should use the versions that match the CCDG workflows at MGI. Are they using 2.9.0 or 2.4.1?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+RUN mkdir /opt/picard/ \
+    && wget https://github.com/broadinstitute/picard/releases/download/2.9.0/picard.jar \
+    && mv picard.jar /opt/picard/
+
+#################
+#Sambamba v0.6.4#
+#################
+
+RUN mkdir /opt/sambamba/ \
+    && wget https://github.com/lomereiter/sambamba/releases/download/v0.6.6/sambamba_v0.6.6_linux.tar.bz2 \
```
Comment (jasonwalker80) [inline Dockerfile]: Should this be v0.6.4?

Comment (tmooney) [reply Dockerfile]: Either the header or the `wget` should definitely change!

File: Dockerfile (hunk 1 above)
Comment (tmooney) [reply Dockerfile]: They are using 2.4.1. I'd copied this one from the rnaseq image I made, which used the latest versions of everything.

File: Dockerfile (hunk 2 above)
Comment (jasonwalker80) [reply Dockerfile]: I think 0.6.4 is the same version as the CCDG workflow.

**Comment by tmooney** [comment]
Updated to downgrade sambamba and Picard.

**Review by jasonwalker80 — APPROVED** [review]
+2

**Pull request merged**